src/graph.rs

```diff
@@ -848,6 +882,13 @@ impl<'a> ModuleEntryIterator<'a> {
                   ));
                 }
               }
+              for import in &dep.imports {
+                if !import.is_dynamic {
```

Comment:
I think this should be:

```suggestion
                if self.follow_dynamic || !import.is_dynamic {
```

I just realized the above on line 872 is wrong and should also be similar.

Reply:
Note to correct https://github.com/denoland/deno/blob/f917d2e2c10e0a94e564a9016217e7ce27c8bbee/cli/tools/vendor/build.rs#L88 to false

Reply:
I changed that one to `true` because I think we should validate dynamic imports for vendoring.

Reply:
Okay I thought this had been masking errors that would now be exposed but it only applies to resolution errors. I can see why missing modules should be caught in advance for vendoring

Reply:
That said, I guess that would prevent vendoring of the scenario you described yesterday where someone might conditionally load some plugin code or something. Maybe instead it should be warnings.